example_parlai_internal/agents/parrot/parrot.py

```diff
@@ -0,0 +1,6 @@
+from parlai.core.torch_agent import TorchAgent, Output
+
+class ParrotAgent(TorchAgent):
+    def eval_step(self, batch):
+        # for each row in batch, convert tensor to string
+        return Output([str(row) for row in batch.text_vec])
```

Comment:
? So it outputs a string of numbers? You don’t want to use vec2txt?

Reply:
This is exactly what's in the quick start. I can always change the quick start as well, but I just copy-pasted

Reply:
I'm so confused. lol Yes, let's fix the quickstart. This parrot agent should get 100% on `-t integration_tests`